src/lazy/system_reader.rs

```diff
@@ -1087,11 +1096,11 @@ mod tests {
         // This directive defines two more.
         assert_eq!(new_macro_table.len(), 2 + MacroTable::NUM_SYSTEM_MACROS);
         assert_eq!(
-            new_macro_table.macro_with_id(3),
+            new_macro_table.macro_with_id(4),
             new_macro_table.macro_with_name("seventeen")
         );
         assert_eq!(
-            new_macro_table.macro_with_id(4),
+            new_macro_table.macro_with_id(5),
```

Comment:
> Did these change because of the addition of annotate?
Yep. [Matt rightly pointed out](https://github.com/amazon-ion/ion-rust/pull/801#discussion_r1712906209) that I should use an offset from `NUM_SYSTEM_MACROS` to prevent future breakage. I'm going to do that for #807.

> The comment on new line 1095 is now out of date though, yes?
Yeah, good catch.